### PR TITLE
export task payment as csv

### PR DESCRIPTION
**Pull request by terryli0095**
Be able to export submission payments as csv 
Screen Shot 2022-07-19 at 11 44 56 PM

### CODE REVIEW

File: wondrous-app/components/Settings/Payouts/exportSubmissionPaymentCsv.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  }
+
+  paymentsArray.forEach((paymentData) => {
+    const assigneeUsername = paymentData.username || '';
+    const wallet = paymentData?.recipientAddress || '';
+    const paymentAmount = paymentData?.amount || '';
+    const tokenSymbol = paymentData?.tokenSymbol || '';
+
+    let newRow;
+    if (exportCSVType === EXPORT_PAYMENT_CSV_TYPE.UTOPIA) {
```
Comment (terryli0095) [inline wondrous-app/components/Settings/Payouts/exportSubmissionPaymentCsv.ts:42]: pretty adhoc logic here but not much better way

File: wondrous-app/components/Settings/Payouts/SubmissionPaymentCSVModal.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  CreateLayoutsModalItem,
+  CreateLayoutsModalItemContainer,
+  CreateLayoutsModalItemTitle,
+  CreateLayoutsModalItemTitleBlock,
+  CreateLayoutsModalTitle,
+} from 'components/CreateEntity/styles';
+import UtopiaIcon from 'components/Icons/utopiaIcon';
+import ParcelIcon from 'components/Icons/parcelIcon';
+
+export const EXPORT_PAYMENT_CSV_TYPE = {
```
Comment (terryli0095) [inline wondrous-app/components/Settings/Payouts/SubmissionPaymentCSVModal.tsx:20]: imported this from constant in another commit

File: wondrous-app/components/Settings/Payouts/SubmissionPaymentCSVModal.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  UTOPIA: 'utopia',
+  PARCEL: 'parcel',
+  PLAIN: 'plain',
+};
+
+export const ENTITIES_UI_ELEMENTS = {
```
Comment (terryli0095) [inline wondrous-app/components/Settings/Payouts/SubmissionPaymentCSVModal.tsx:26]: moved these to constants inanother pr

File: wondrous-app/components/Settings/Payouts/exportSubmissionPaymentCsv.ts [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  let headers = [];
+  if (exportCSVType === EXPORT_PAYMENT_CSV_TYPE.UTOPIA) {
+    headers = ['Name', 'Wallet', 'Amount', 'Denominated in', 'Pay-out Token'];
+  } else if (exportCSVType === EXPORT_PAYMENT_CSV_TYPE.PARCEL) {
+    headers = ['Name(Optional)', 'Address/ENS', 'Amount', 'Token Address/Token Symbol', 'Symbol'];
+  } else if (exportCSVType === EXPORT_PAYMENT_CSV_TYPE.PLAIN) {
+    headers = ['Username', 'Address/ENS', 'Amount', 'Token Address', 'Token Name', 'Symbol', 'Chain', 'Decimals'];
+  }
+  const rows = [[headers]];
+  console.log('unpaidSubmissions', unpaidSubmissions);
```
Comment (Lamperoyge) [inline wondrous-app/components/Settings/Payouts/exportSubmissionPaymentCsv.ts:14]: probably need to remove this

File: wondrous-app/components/Settings/Payouts/SubmissionPaymentCSVModal.tsx (hunk 3 above)
Comment (Lamperoyge) [reply wondrous-app/components/Settings/Payouts/SubmissionPaymentCSVModal.tsx:26]: nice

**Pull request merged**